media/css/firefox/hello/index.less

```diff
@@ -141,7 +122,9 @@ main {
 }
 
 #powered-by {
-    .font-size(@smallFontSize);
+    .font-size(@baseFontSize);
+    .open-sans-italic;
```

Comment:
I saw this mixin was added recently but I'm not sure it's needed. Is this better than just <code>font-style: italic</code>? The mixin explicitly declares the font-family but as far as I can tell that's already inherited.